src/nnet3/nnet-simple-component.h

```diff
@@ -880,6 +929,218 @@ class NaturalGradientPerElementScaleComponent: public PerElementScaleComponent {
 };
 
 /**
+ * ConvolutionalComponent implements 2d-convolution.
+ * It uses 3D filters on 3D inputs, but the 3D filters hop only over
+ * 2 dimensions as it has same size as the input along the 3rd dimension.
+ * Input : A matrix where each row is a  vectorized 3D-tensor.
+ *        The 3D tensor has dimensions
+ *        x: (e.g. time)
+ *        y: (e.g. frequency)
+ *        z: (e.g. channels like features/delta/delta-delta)
+ *
+ *        The component supports input vectorizations of type zyx and yzx.
+ *        The default vectorization type is zyx.
+ *        e.g. for input vectorization of type zyx the input is vectorized by
+ *        spanning axes z, y and x of the tensor in that order.
```

Comment:
this is not really clear to me-- if you explain it in terms of which dimension has the largest stride, it might be clearer.
E.g. in zyx vectorization, does z have a stride of one, or does x?